examples/ts-react-chat/src/routes/api.tanchat.ts

```
@@ -7,15 +7,21 @@ import {
+import { anthropicText, createAnthropicChat } from '@tanstack/ai-anthropic'
+import { createGeminiChat, geminiText } from '@tanstack/ai-gemini'
+import {
+  createGeminiTextInteractions,
+  geminiTextInteractions,
+} from '@tanstack/ai-gemini/experimental'
+import { createOpenRouterText, openRouterText } from '@tanstack/ai-openrouter'
+import { createGrokText, grokText } from '@tanstack/ai-grok'
+import { createGroqText, groqText } from '@tanstack/ai-groq'
 import { bedrockText } from '@tanstack/ai-bedrock'
+import { byokMissing, getByokKey } from '@tanstack/ai-byok/server'
+import { BYOK_PROVIDER_MAP, byokIdForProvider } from '@/lib/byok-config'
```

ESLint

[error] `@/lib/byok-config` import should occur after type import of `@tanstack/ai`

(import/order)